### DISCUSSION TITLE
React 19 - what is the React Hook Form look line then

### DISCUSSION THREAD

**Discussion by totaland** [react-hook-form/react-hook-form#11585] [General]
Hi team, 

The React team came up with new API such as: "use", "useFormState", "useFormStatus". What will be the future road map for React Hook Form, will it still be relevant?

**Comment by Moshyfawn** [comment]
These new form APIs handle some of the form state that RHF typically manages for you, but that's not the only thing RHF is for. For the `use` API, RHF already supports `use` for the async default values, so we're pretty much set there as well. We might experiment with some bits and pieces of integrating the React provided states, but I don't think we've thought about it too much yet.
